torch/testing/_internal/common_utils.py

```diff
@@ -1894,11 +1892,34 @@ def __enter__(self):
     def __exit__(self, *args):
         pass
 
-def _assertGradAndGradgradChecks(test_case, apply_fn, inputs, check_batched_grad=False):
+
+def gradcheck(fn, inputs, **kwargs):
+    # Wrapper around gradcheck that enables certain keys by default
```

Comment:
nit: It may be helpful to the reader to provide more context about why we want this. E.g. the reason behind having a testing-internal version of gradcheck is because autograd.gradcheck is a public-facing API. All PyTorch devs doing testing should use the testing-internal version of gradcheck instead of autograd.gradcheck